pkg/k2/constructs/construct_evaluator.go

```diff
@@ -1,28 +1,23 @@
 package constructs
 
 import (
-	"bytes"
 	"context"
 	"errors"
 	"fmt"
+	"github.com/klothoplatform/klotho/pkg/async"
+	template2 "github.com/klothoplatform/klotho/pkg/k2/constructs/template"
```

Comment:
Since text/template isn't used, this can just be `template` and not aliased, right?

Also, nit: any way to configure your IntelliJ to sort these? I believe the Go language server uses the same as [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) (an official tool).